samples/transient-clients/Core_6/StockEvents/StockTick.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+
+public class StockTick
```

Comment:
This allows the Client (non-NSB) to share the message type with the NServiceBus endpoints.